src/components/MoneyRequestReportView/MoneyRequestReportView.tsx

```diff
@@ -161,6 +147,10 @@ function MoneyRequestReportView({report, policy, reportMetadata, shouldDisplayRe
         );
     }
 
+    // Special case handling a report that is a single transaction thread
```

Comment:
Better not to mention the single here as it might be more confusing than helpful
```suggestion
    // Special case handling a report that is a transaction thread
```